assets/messages.yml

```
@@ -590,6 +590,23 @@ messages:
 
         %quick_links%
       fillname: []
+  not-reopen-ar:
+    - project:
+        - MC
+        - MCTEST
+        - MCPE
+        - MCAPI
+        - MCL
+        - MCD
+        - MCE
+        - BDS
+        - REALMS
+        - WEB
```

```suggestion
        - bds
        - mc
        - mcapi
        - mcd
        - mce
        - mcl
        - mcpe
        - mctest
        - realms
        - web
```
Lowercase them and use same (alphabetical) order as for other messages.